designs/ClientDevProcess.md

```diff
@@ -0,0 +1,145 @@
+
+### Pipelines
+
+These are the columns on the board, and each one has a specific purpose, described below. Note that epics are handled slightly different, as outlined in [Epic Tracking](#epic-tracking):
+
+#### New Issues
+
+This is the default pipeline/column for incoming issues. Whenever a new issue is created in NuGet/Home, a card is added here to represent that story.
+
+As the team works to whittle down the list of issues in the repository, this pipeline will become empty except for un-triaged issues. From here, one of three things will happen to an issue:
+
+1. It will be moved to the Icebox
+2. It will be moved to the Backlog
+3. It will be closed
+
+#### Icebox
+
+The Icebox is for issues the team thinks are worth doing, doesn't foresee having time to do themselves. Generally, these issues should also be tagged as `help wanted` and, if appropriate, `good first issue`. Issues from the Icebox can progress in two ways:
+
+1. If the Backlog is exhausted
+2. The issue is closed because it's no longer relevant
+
+#### Backlog
+
+The Backlog is the overall task list for the Client Team. This is the work the team intends to be doing itself in the current sprint or in the next two sprints. Anything else goes back in the Icebox.
+
+The general intention with the Backlog is that when a team member has completed all their work, they should be able to pull off something near the top of the Backlog and start working on it.
+
+Once something is in the Backlog, it can also be assigned a Sprint. This allows us to filter the Backlog into a Sprint Backlog by filtering by Sprint! Sprint Backlog items take priority over other Backlog items and should generally be assigned to Sprints during Sprint Planning, sized in such a way that it'll be just enough for 2 weeks of work.
+
+As items enter the Backlog, they should be prioritized as well as possible, with more important issues bubbling to the top, and, if high priority, marked as such, which will pin it to the top.
+
+All issues in the Backlog should be estimated, with each Story Point being roughly 1 hour, with a strong preference towards increments of 5. Estimates include review and review fixes time. Epics should not be directly estimated, and should instead rely on their sub-tasks' estimates.
+
+The Backlog can further be filtered into three main buckets, using GitHub Labels:
+
+1. Quality - test improvements, bugfixes
+2. Engineering - engineering process improvements like CI and other process optimizations
+3. Product - Issues associated with OKRs
+
+During a Sprint, the first two weeks should be spent on Product Backlog issues, according to team assignments. Starting week 3, or earlier if someone's Product tasks are done, the Client Team will start pulling things out of the Quality and Engineering Backlogs.
+
+Issues in the Backlog should not be assigned to anyone. They only get assigned when moved to In Progress.

```

Comment:
I imagine this involves removing ourselves as assignees for many issues? :)

Reply:
Eh. I figure we'll do it piecemeal, or people can just do it over time on things they're assigned to. I don't think it'll be that much work to get 80% of the benefit of this.

Reply:
I keep some issues assigned to myself, because at some point I had a plan to fix them. 
Sometimes I do get to some old issues assigned to me, but certainly won't happen with all of them :)